server/routes/api/chart.py

```diff
@@ -544,7 +544,9 @@ def choropleth_data(dcid):
             sv_denom_result = {
                 'date': sv_date,
                 'data': geo_data,
-                'numDataPoints': num_data_points
+                'numDataPoints': num_data_points,
+                # TODO (chejennifer): exploreUrl should link to choropleth tool once the tool is ready
+                'exploreUrl': build_url(dcid, [sv])
```

Comment:
don't forget to set is_scaled=true if there is a denominator

Reply:
it doesn't look like there's anywhere to set that?

Reply:
oh sorry! i thought i had merged #488